requirements.txt

```diff
@@ -4,3 +4,5 @@ lit
 numpy<2.0.0
 xdsl @ git+https://github.com/xdslproject/xdsl.git@0d7eae3a1c9ee128986d6729ffc1d49786f7819d
 tensorflow-cpu==2.15.0
+pandas
+matplotlib
```

Comment:
can we put this in an optional requirements please?

Reply:
done!